Tests/test_content.py

```diff
@@ -902,6 +912,38 @@ def manage_tests(tests_settings):
             is_build_passed_file.write('Build passed')
 
 
+def add_pr_comment(comment):
+    token = os.environ['CONTENT_GITHUB_TOKEN']
+    branch_name = os.environ['CIRCLE_BRANCH']
+    sha1 = os.environ['CIRCLE_SHA1']
+
+    query = '?q={}+repo:demisto/content+org:demisto+is:pr+is:open+head:{}+is:open'.format(sha1, branch_name)
+    url = 'https://api.github.com/search/issues'
+    headers = {'Authorization': 'Bearer ' + token}
+    try:
+        res = requests.get(url + query, headers=headers, verify=False)
+        handle_github_errors(res)
+
+        res = res.json()
+        if res and res.get('total_count', 0) == 1:
+            issue_url = res['items'][0].get('comments_url') if res.get('items', []) else None
+            if issue_url:
+                res = requests.post(issue_url, json={'body': comment}, headers=headers, verify=False)
+                handle_github_errors(res)
+        else:
+            raise Exception('There is more then one open pull request for branch {}.'.format(branch_name))
+
+    except Exception as e:
+        print_error('Add pull request comment failed: {}'.format(e))
+
+
+def handle_github_errors(response):
+    res_dict = response.json()
+    if response.ok:
+        return res_dict
+    raise Exception(res_dict.get('message'))
```

Comment:
don't raise an error, this is a "nice to have" feature, it should print a warning and continue